dla_cnn/desi/write_as_tf_file.py

```diff
@@ -0,0 +1,25 @@
+from dla_cnn.desi.training_sets import split_sightline_into_samples,select_samples_50p_pos_neg
+from dla_cnn.desi.preprocess import label_sightline
+import tensorflow as tf
+
+def write_as_tf_file(sightline, output_path):
+    '''
+    sightline:dla_cnn.data_model.Sightline
```

Comment:
Add a short note on what this method does.

And define `output_path`

Reply:
got it